data/versions.yml

```diff
@@ -2,4 +2,35 @@
 main: "1.25"
 
 # The version of Istio currently documented in preliminary.istio.io
-preliminary: "1.26"
+preliminary: "1.27"
```

Comment:
You should probably rebase this PR against the current master.  Let me know if you need help doing that.